Sources/CMakeLists.txt

```diff
@@ -12,12 +12,29 @@
 ##
 ##===----------------------------------------------------------------------===##
 
-add_subdirectory(_FoundationCShims)
-
-# Disable the macro build on Windows until we can correctly build it for the host architecture
-if(NOT CMAKE_SYSTEM_NAME STREQUAL Windows)
-	add_subdirectory(FoundationMacros)
+include(ExternalProject)
+if(CMAKE_HOST_WIN32)
+  set(_SwiftFlags -DCMAKE_Swift_FLAGS="-use-ld=lld")
```

Comment:
Since it's flags for the macro build, would you mind making that clearer in the variable name?
e.g. `_SwiftMacroFlags` or something like that? Thanks.

Reply:
Done